lib/morpheus.gi

```diff
@@ -1959,6 +1959,26 @@ local A;
   return A;
 end);
 
+#############################################################################
+##
+#M  AutomorphismGroup(<G>) . . abelian case
+##
+InstallMethod(AutomorphismGroup,"test abelian",true,[IsGroup and IsFinite],
+  SIZE_FLAGS(WITH_HIDDEN_IMPS_FLAGS(FLAGS_FILTER(
+    IsSolvableGroup and IsFinite))),
```

Comment:
I don't understand this, what's its purpose? Note that for me, it returns the value 50 -- t same as `RankFilter(IsSolvableGroup and IsFinite)`.

Reply:
@fingolfin 
At some time in the past SIZE_FLAGS(WITH_HIDDEN_IMPS_FLAGS(FLAGS_FILTER was the required construction to get the value of a filter. It still persists in a few places in the library.
If this ugly constrcut has been superseded by RankFilter all the better and I'd be happy to change it.

Reply:
It has - sometimes in the 90ies, in fact ;-)

You are right, thought, several `SIZE_FLAGS(WITH_HIDDEN_IMPS_FLAGS(FLAGS_FILTER(foo)))` constructs survive, a few of them apparently were left to allow the (now removed) "completion files" to work. I'll make a PR to get rid of as many as I can.

Reply:
Done, see PR #1840

Reply:
It has changed in the 90s? Next you tell me that Helmut Kohl is not chancellor any longer!

I've changed it as well. Do I now wait for you to clear the review, or do I dismiss it?